app/build.gradle.kts

```diff
@@ -42,8 +42,12 @@ dependencies {
     implementation(libs.retrofit)
     implementation(libs.retrofit.gson)
     implementation(libs.picasso)
+    implementation(libs.room.runtime)
+    implementation(libs.room.ktx)
     implementation(libs.androidx.recyclerview)
+    implementation ("androidx.lifecycle:lifecycle-viewmodel-ktx:2.8.4")
```

Comment:
keep consistency, this is not the way to declare dependencies

Reply:
Fixed in next comit